CHANGELOG.md

```diff
@@ -7,6 +7,15 @@ This project adheres to the Node [default version scheme](https://docs.npmjs.com
 
 ## [next-version]
 
+### Added
+
+### Changed
+- Internal: Remove unused dependencies and scripts from `package.json`
+
+### Fixed
+- UI: Accessibility - Focus is at document start
+- UI: Camera not detected on Safari 13.1 on iOS 13.4.1, MacOSX 10.15.4
```

Comment:
[Minor] OS naming:
- earlier: Mac OS X
- now: macOS

I believe you're referring to earlier version here.

Reply:
Ah yes. You're right!